aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,86 @@ uid: mvc/controllers/actions
+Controllers usually inherit from [Controller](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.controller), although this is not required. Deriving from `Controller` provides access to three categories of helper methods designed to return specific response types:
 
-**[View](../views/index.md)**
+#### 1. An empty response body
 
-Returns a view that uses a model to render HTML. Example: `return View(customer);`
+No `Content-Type` HTTP response header is included, since the response body lacks content to describe.
 
-**HTTP Status Code**
+There are two response types within this category: Redirect and HTTP Status Code. The Redirect response type differs from the HTTP Status Code type primarily in the addition of a `Location` HTTP response header.
 
-Return an HTTP status code. Example: `return BadRequest();`
+* **Redirect**
 
-**Formatted Response**
+    This type returns a redirect to an action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction`, or `RedirectToRoute`). For example, `return RedirectToAction("Complete", new {id = 123});` redirects to `Complete`, passing an anonymous object.
 
-Return `Json` or similar to format an object in a specific manner. Example: `return Json(customer);`
+* **HTTP Status Code**
 
-**Content negotiated response**
+    This type returns an HTTP status code. A couple helper methods of this type are `BadRequest` and `NotFound`. For example, `return BadRequest();` produces a 400 status code when executed.
 
-Instead of returning an object directly, an action can return a content negotiated response (using `Ok`, `Created`, `CreatedAtRoute` or `CreatedAtAction`). Examples: `return Ok();` or `return CreatedAtRoute("routename",values,newobject);`
+#### 2. A non-empty response body with a predefined content type
 
-**Redirect**
+Most of these response types include a `ContentType` property, allowing you to set the `Content-Type` response header to describe the response body.
 
-Returns a redirect to another action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction` or `RedirectToRoute`). Example: `return RedirectToAction("Complete", new {id = 123});`
+There are two response types within this category: [View](xref:mvc/views/overview) and [Formatted Response](xref:mvc/models/formatting).
 
-In addition to the methods above, an action can also simply return an object. In this case, the object will be formatted based on the client's request. Learn more about [Formatting Response Data](../models/formatting.md)
+* **View**
 
-### Cross-Cutting Concerns
+    This type returns a view which uses a model to render HTML. For example, `return View(customer);` passes a model to the view for data-binding.
+
+* **Formatted Response**
+
+    This type returns JSON or a similar data exchange format to represent an object in a specific manner. For example, `return Json(customer);` serializes the provided object into JSON format.
```

Comment:
Might want to mention the `*File(...)` methods (`File(...)`, `PhysicalFile(...)` and `VirtualFile(...)`). All of these accept a `string contentType` parameter and none do content negotiation.